RELEASE.md

```diff
@@ -161,6 +162,8 @@ If this step fails, the deephaven-server wheel from the "Upload Artifacts" step
 The "Publish pydeephaven to PyPi" uploads the pydeephaven wheel to [PyPi](https://pypi.org/project/pydeephaven/).
 If this step fails, the pydeephaven wheel from the "Upload Artifacts" step can be uploaded manually.
 
+There is no automatic uploads for the pydeephaven-ticking wheels to [PyPi](https://pypi.org/project/pydeephaven-ticking/).
```

Comment:
Isn't that the `Publish pydeephaven-ticking to PyPi` action that you're adding with this PR? Not clear why pydeephaven-ticking isn't automatic but the others are based on this comment.

Reply:
Forgot to update, will do.